### ISSUE TITLE
Header doc comments are out of date

### ISSUE THREAD

**Issue by Ashton-W** [tomaz/GBCli#15]
The header comments are out of date compared to the Readme. The code samples are now wrong as the method signatures have changed.

See [GBCli/src/GBCommandLineParser.h](https://github.com/tomaz/GBCli/blob/master/GBCli/src/GBCommandLineParser.h)

**Comment by tomaz** [comment]
Indeed. It's probably simplest to just remove examples from source code comments - one less thing to maintain... It's not very high on my priority list though :)
